docker/openjdk-11/Dockerfile

```diff
@@ -0,0 +1,41 @@
+
+FROM openjdk:11.0.7-jre-slim-buster
```

Comment:
Should this replace the existing docker/Dockerfile which still seems to be present?

Reply:
Good catch.  It was sitting in another CL.